doc/onboarding/model/form.md

```diff
@@ -0,0 +1,30 @@
+# `form` State
+
+## Definition
+State of a form that includes its values but also :
+ - required field or
+ - ability to submit or not all the form.
+
+```
+{
+    <boolean> disabled: false
```

Comment:
@gregorylegarec, @CPatchane : l think that form is disabled when password complexity is not "green"; we have to check with @jsilvestre